src/v/v8_engine/script.h

```diff
@@ -44,6 +45,7 @@ class script_exception final : public std::exception {
 class script {
     // Timeout for run script in first time for initialization
     static constexpr std::chrono::milliseconds _first_run_timeout_ms{500};
+    static constexpr uint64_t _max_old_gen_size{10_MiB};
```

Comment:
How is this value chosen? Should it be configurable?

Reply:
With this value test works. 
I want to return to this settings in data_policy testing

Reply:
I think Ben is asking why `10MiB`? Was this the value older versions of v8 were using before it was exposed?

Reply:
My ans is It is the most smallest value when test works(can create local context for v8).